### PR TITLE
Update gradle build to publish

### PR DESCRIPTION
**Pull request by kierans**
This updates the Gradle build with the necessary steps to publish to OSSHR Nexus for #5 

I've tested the signing step of the build using a dummy GPG key.

The major impact to consumers is the changing of the `groupId` to be `au.com.wpay`.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -20,4 +20,35 @@ android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
 
-org.gradle.configureondemand=true
\ No newline at end of file
+org.gradle.configureondemand=true
+
+GROUP=au.com.woolworths.village.sdk
```
Comment (applimatix) [inline gradle.properties]: @JamesMBligh do we have any plans to remove the village reference from the SDK's or is the village name living on an some other capacity?

**Review by applimatix — APPROVED** [review]
Looks good - my only point is my comment to James about the village reference in the GroupID as we probably want to get this right from day 1

**Comment by applimatix** [comment]
haha I spoke too soon

File: gradle.properties (hunk 1 above)
Comment (kierans) [reply gradle.properties]: I'm wanting it to remove it when we rewrite the SDK to follow the same approach as we did in the JS SDK rewrite. We got rid of the generated code which made it a lot easier to develop/maintain.

**Pull request merged**